ansible/roles/test/tasks/pfc_asym.yml

```diff
@@ -67,16 +67,6 @@
 
   # Get lossless and lossy priorities
 
-  - name: Get port QOS map key
-    command: docker exec -i database redis-cli --raw -n 4 KEYS *PORT_QOS_MAP*
-    register: out
-  - set_fact: port_qos_key={{out.stdout}}
-
-  - name: Get lossless priorities
-    command: docker exec -i database redis-cli -n 4 HGET "{{port_qos_key}}" "pfc_enable"
-    register: out
-  - set_fact: lossless_priorities={{out.stdout.split(",")}}
```

Comment:
This method to get lossless_priorities is still valid if {{port_qos_key}} is updated to a single physical port